zigpy/application.py

```diff
@@ -1221,6 +1221,9 @@ def _device(self) -> zigpy.device.Device:
         return self.get_device(ieee=self.state.node_info.ieee)
 
     def _persist_coordinator_model_strings_in_db(self) -> None:
+        if 1 not in self._device.endpoints:
+            self._device.add_endpoint(1)
+
         cluster = self._device.endpoints[1].add_input_cluster(
             zigpy.zcl.clusters.general.Basic.cluster_id
         )
```

Comment:
No, it does not, however I believe new endpoints can be implemented on the application level.

If you feel uncomfortable adding an endpoint, we could instead pick the first existing one like this:

```suggestion
        cluster = self._device.non_zdo_endpoints[0].add_input_cluster(
            zigpy.zcl.clusters.general.Basic.cluster_id
        )
```

Please advise.

Reply:
Perhaps it would be best for the XBee implementation to call `await self.register_endpoints()` within `start_network` and then to implement `register_endpoint` to just call `add_endpoint`. This is more of a radio library bug more than a zigpy bug, in my opinion.

Reply:
Is endpoint 1 mandatory?

Reply:
Yes. Zigpy registers endpoints explicitly on startup: https://github.com/zigpy/zigpy/blob/32a3f050327618a66f88a8b3d77306804826a9cd/zigpy/application.py#L701

Reply:
Thanks! I will check.